README.md

```diff
@@ -63,7 +112,178 @@ And finally you can start the VM:
+ Code on VM          | /opt/borrower_frontend
+ Default port        | 0.0.0.0:9000
+
+### Conveyancer Frontend
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | conveyancer_frontend
+ Code                | [charges-conveyancer-frontend](https://github.com/LandRegistry/charges-conveyancer-frontend)
+ Infrastructure code | [charges-conveyancer-frontend/puppet/borrower_frontend](https://github.com/LandRegistry/charges-conveyancer-frontend/tree/master/puppet/conveyancer_frontend)
+ Development Host    | http://conveyancer_frontend.dev.service.gov.uk
+ Code on VM          | /opt/conveyancer_frontend
+ Default port        | 0.0.0.0:9040
+
+### Deed API
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | deed_api
+ Code                | [charges-deed-api](https://github.com/LandRegistry/charges-deed-api)
+ Infrastructure code | [charges-deed-api/puppet/deed_api](https://github.com/LandRegistry/charges-deed-api/tree/master/puppet/deed_api)
+ Development Host    | http://deed_api.dev.service.gov.uk
+ Code on VM          | /opt/deed_api
+ Default port        | 0.0.0.0:9012
+
+### Case API
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | case_api
+ Code                | [charges-case-api](https://github.com/LandRegistry/charges-case-api)
+ Infrastructure code | [charges-case-api/puppet/case_api](https://github.com/LandRegistry/charges-case-api/tree/master/puppet/case_api)
+ Development Host    | http://case_api.dev.service.gov.uk
+ Code on VM          | /opt/case_api
+ Default port        | 0.0.0.0:9070
+
+### Scribe
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | scribe
+ Code                | [charges-scribe](https://github.com/LandRegistry/charges-scribe)
+ Infrastructure code | [charges-scribe/puppet/scribe](https://github.com/LandRegistry/charges-scribe/tree/master/puppet/scribe)
+ Development Host    | http://scribe.dev.service.gov.uk
+ Code on VM          | /opt/scribe
+ Default port        | 0.0.0.0:9010
+
+### Verifudge
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | verifudge
+ Code                | [charges-verifudge](https://github.com/LandRegistry/charges-verifudge)
+ Infrastructure code | [charges-verifudge/puppet/verifudge](https://github.com/LandRegistry/charges-verifudge/tree/master/puppet/verifudge)
+ Development Host    | http://verifudge.dev.service.gov.uk
+ Code on VM          | /opt/verifudge
+ Default port        | 0.0.0.0:9080
+
```

Comment:
It is but it's not really an app, so I didn't want to put it here. Any ideas where I should mention it?